cypress.config.ts

```diff
@@ -0,0 +1,20 @@
+import { defineConfig } from 'cypress'
```

Comment:
Do we still need this?

Reply:
that just a temporary added file when i run cypress, the whole folder will be remove in next coming pr, prefer to made the change in the next pr 🙏🏻 (reduce merge conflict)

Reply:
Should we remove this?

Reply:
As per our conversation, we will remove this file in the next coming up pr, because there are still some reference of cypress in other example project